site/docs/ch/mysql-install.md

```diff
@@ -0,0 +1,252 @@
+### 物理机部署mysql
+
+
+
+>  mysql链接
+
+```
+ http://nos-yq.126.net/innosql-release/mysql-5.7.20-v3e-linux-x86_64.tar.gz
```

Comment:
Do all user can access this url?
Does a open source MySQL version is more suitable for open source users?

Reply:
This url is public